openlibrary/core/bookshelves.py

```diff
@@ -230,6 +321,9 @@ def get_users_logged_books(
         from openlibrary.plugins.worksearch.code import run_solr_query
         from openlibrary.plugins.worksearch.schemes.works import WorkSearchScheme
 
+        logger.info("BOOKSHELF: %s" % bookshelf_id)
+        # Sets the function to fetch editions as well, if not accessing the Want to Read shelf.
+        editions: bool = bookshelf_id != 1
```

Comment:
Rename variable to feel more bool-y

```suggestion
        show_editions: bool = bookshelf_id in != 1
```

Reply:
It has been thoroughly bool-ed, now. Thanks for all the advice with variable names, it shouldn't be as necessary in the future (I hope!)